assets/src/edit-story/components/elementLink/frame.js

```diff
@@ -86,7 +86,7 @@ function WithLink({ element, active, children, anchorRef }) {
         placement={'top'}
         spacing={spacing}
       >
-        {link && (
+        {link?.url?.length && (
```

Comment:
If length is 0, this will actually output `0` to the page. I think this is better/safer: `Boolean(link?.url) && ...`.